assets/js/invenio_app_rdm/frontpage/RecordResultsListItem.js

```diff
@@ -0,0 +1,104 @@
+// This file is part of CDS RDM
+// Copyright (C) 2022 CERN.
+//
+// CDS RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import _get from "lodash/get";
+import _truncate from "lodash/truncate";
+import _upperCase from "lodash/upperCase";
+import React from "react";
+import { Grid, Label, Image, Container } from "semantic-ui-react";
+import PropTypes from "prop-types";
+
+export const RecordResultsListItem = ({ result }) => {
+  const accessStatusId = _get(result, "ui.access_status.id", "open");
+  const accessStatus = _get(result, "ui.access_status.title_l10n", "Open");
+  const accessStatusIcon = _get(result, "ui.access_status.icon", "unlock");
+  const descriptionStripped = _get(
+    result,
+    "ui.description_stripped",
+    "No description"
+  );
+
+  const publicationDate = _get(
+    result,
+    "ui.publication_date_l10n_long",
+    "No publication date found."
+  );
+  const resourceType = _get(
+    result,
+    "ui.resource_type.title_l10n",
+    "No resource type"
+  );
+  const title = _get(result, "metadata.title", "No title");
+  const version = _get(result, "ui.version", null);
+  const community = _get(
+    result,
+    "expanded.parent.communities.default.metadata.title",
+    null
+  );
+  const descriptionLength = community ? 3000 : 2500;
+
+  // Derivatives
+  const viewLink = `/records/${result.id}`;
+  return (
+    <Container>
+      <Grid columns={2}>
+        <Grid.Row stretched>
+          <Grid.Column width={4}>
+            <Image
```

Comment:
Minor: We should use the `Image` component from react-invenio-forms (check it [here](https://github.com/inveniosoftware/react-invenio-forms/blob/master/src/lib/elements/Image.js#L16)) instead of the one from semantic-ui-react.